advanced_function/task3.js

```
@@ -0,0 +1,49 @@
+function divide(numerator,denominator){
+    if (denominator===0){
+        console.log("Validation message: denominator is null")
+    }
+    else {
+
+    if ((isNaN(denominator)) || (isNaN(numerator))){
+        console.log("Validation message: denominator or numerator is not a number")
```

throw error instead of printing